source/common/json/json_streamer.h

```diff
@@ -331,32 +351,36 @@ class Streamer {
    */
   void addSanitized(absl::string_view prefix, absl::string_view token, absl::string_view suffix) {
     absl::string_view sanitized = Json::sanitize(sanitize_buffer_, token);
-    response_.addFragments({prefix, sanitized, suffix});
+    response_.add(prefix, sanitized, suffix);
+  }
+
+  void addString(absl::string_view str) {
```

Comment:
can we be more explicit by calling this `addSanitizedWithQuotes` or something like that?

addString does not make it obvious that sanitizaiton and quoting will be happening.

Also, needs doc.

And why not implement by calling `addSanitized("\"", str, "\"");` ?

Reply:
I will also argue for the naming. Most json lib will provide a similar method to add a string to the buffer/object. When users see this method in a json related lib. It's clear what will happen. And note our Level has provide an `addString`.

> And why not implement by calling addSanitized("\"", str, "\""); ?

Good point. Will update it.